loader/integration.go

```diff
@@ -55,3 +56,31 @@ func GetLatestPipelineRunForSnapshotAndScenario(adapterClient client.Client, ctx
 
 	return nil, err
 }
+
+// RemoveFinalizerFromAllIntegrationPipelineRunsOfSnapshot fetches all the Integration
+// PipelineRuns associated with the given Snapshot and each of the IntegrationTestScenarios.
+// After fetching them, it removes the finalizer from the PipelineRun, and returns error if any.
+func RemoveFinalizerFromAllIntegrationPipelineRunsOfSnapshot(adapterClient client.Client, logger helpers.IntegrationLogger, ctx context.Context, snapshot applicationapiv1alpha1.Snapshot, finalizer string) error {
```

Comment:
This func is in loaders module, but it updates things not loading them. IMO it shouldn't be here

Reply:
Should stay in helpers IMO, it will also remove new dependency on helpers module here

Reply:
You're right, I've now moved it to helpers/integration.go file.